src/css/custom.css

```diff
@@ -78,13 +78,13 @@ table {
 /* Make "Buy Now" link styled as button */
 .buy-now-nav-item {
   outline: none;
-  background: var(--theme-color-dark-navy-blue) !important;
+  background: var(--theme-color-stormy-blue) !important;
```

Comment:
We can remove `!important`

Reply:
If we remove it it doesn't work properly for mobile view :( 

https://github.com/vazco/forminer/assets/89641881/9c21572e-0a87-48ba-a48c-fb0569b702a6

Reply:
My bad, I haven't checked it!